packages/canvas/render/src/page-block-function/methods.ts

```
@@ -0,0 +1,27 @@
+import { reset } from '../data-utils'
+import { parseData } from '../data-function'
+import type { IFuncType } from '@opentiny/tiny-engine-dsl-vue'
+
+export function useMethods({ getContext, setContext }) {
+  const methods: Record<string, Function> = {}
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)